django_project/ledet_base/templates/landing_page.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'main_base.html' %}
+{% load static from staticfiles %}
+{% load jsonify %}
+
+{% block title %}
```

Comment:
```suggestion
{% block subtitle %}
```

Use subtitle instead